pom.xml

```diff
@@ -99,7 +99,8 @@
         <maven-resources-plugin.version>3.0.1</maven-resources-plugin.version>
         <maven-war-plugin.version>3.0.0</maven-war-plugin.version>
         <frontend-maven-plugin.version>1.3</frontend-maven-plugin.version>
-        <pax-web-extender-whiteboard.version>6.0.6</pax-web-extender-whiteboard.version>
+        <pax-web.version>6.0.6</pax-web.version>
```

Comment:
Do we need to redeclare this here? Or can we just inherit it from brooklyn-server's pom.xml (which it should get from its parent).

Reply:
I should remove it.

I think I was having problems building `brooklyn-ui` independent of the main `apache/brooklyn` project and it was pulling the `brooklyn-server` snapshot from the repo rather than my previously built locally copy.